lib/ansible/constants.py

```diff
@@ -261,6 +261,7 @@ def load_config_file():
 
 # characters included in auto-generated passwords
 DEFAULT_PASSWORD_CHARS = ascii_letters + digits + ".,:-_"
+DEFAULT_DONT_TYPE_FILTERS = get_config(p, 'jinja2', 'dont_type_filters', 'ANSIBLE_DONT_TYPE_FILTERS', ['to_json', 'to_nice_json', 'to_yaml', 'ppretty', 'json'], islist=True )
```

Comment:
The name isn't obvious to me.

How about DEFAULT_AS_IS_FILTERS or similar?

This should be documented too, but I can do that separately if need be.